pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
     "holidays >= 0.13",
     "numpy >= 1.21.0",
     "packaging >= 20.0",
-    "pandas >= 1.4.0",
+    "pandas >= 1.5.0",
```

Comment:
Need to bump this to get `NaturalLanguageToYear` unit tests tor return consistent data types